nav2_bringup/params/nav2_multirobot_params_1.yaml

```diff
@@ -172,47 +179,62 @@ local_costmap:
       height: 3
       resolution: 0.05
       robot_radius: 0.22
-      plugins: ["obstacle_layer", "inflation_layer"]
+      plugins: ["voxel_layer", "inflation_layer"]
       inflation_layer:
         plugin: "nav2_costmap_2d::InflationLayer"
         cost_scaling_factor: 3.0
         inflation_radius: 0.55
-      obstacle_layer:
-        plugin: "nav2_costmap_2d::ObstacleLayer"
+      voxel_layer:
+        plugin: "nav2_costmap_2d::VoxelLayer"
         enabled: True
+        publish_voxel_map: True
```

Comment:
set to false

Reply:
Done in the https://github.com/ros-planning/navigation2/pull/3383/commits/8039ff25d68f2f349660c426f62c8cdeb5479d20